### PR TITLE
33 modify tables

### PR DESCRIPTION
**Pull request by EmmaPeders1**
Removed the “Select”-buttons at the tables and madee entire row clickable instead. 
Added descriptive header.

### CODE REVIEW

File: kegeland-frontend/src/components/DataTable.tsx [hunk 1]
```diff
@@ -32,13 +32,14 @@ import {
...
 } from '@tanstack/react-table';
+import { useNavigate } from 'react-router-dom';
 
 export type DataTableProps<T extends object> = {
   data: T[];
   columns: ColumnDef<T, any>[];
 };
 
-const DataTable = <T extends object>({ data, columns }: DataTableProps<T>) => {
+const DataTable = <T extends { object?: any, id?: string } >({ data, columns }: DataTableProps<T>) => {
```
Comment (anines) [inline kegeland-frontend/src/components/DataTable.tsx]: Why do you use type any here? Could you make a custom type including the types that should be accepted?

File: kegeland-frontend/src/components/DataTable.tsx [hunk 2]
```diff
@@ -54,6 +55,30 @@ const DataTable = <T extends object>({ data, columns }: DataTableProps<T>) => {
     debugHeaders: true,
     debugColumns: false,
   });
+
+  const navigate = useNavigate();
+
+  {/*]
```
Comment (anines) [inline kegeland-frontend/src/components/DataTable.tsx]: Is the commented section to be used later? Or could it be removed?

**Review by anines — CHANGES_REQUESTED** [review]

File: kegeland-frontend/src/components/DataTable.tsx (hunk 1 above)
Comment (EmmaPeders1) [reply kegeland-frontend/src/components/DataTable.tsx]: Could be removed

File: kegeland-frontend/src/components/DataTable.tsx (hunk 2 above)
Comment (EmmaPeders1) [reply kegeland-frontend/src/components/DataTable.tsx]: Just an alternative, will remove

**Review by anines — APPROVED** [review]

**Pull request merged**